.env.example

```
@@ -1,6 +1,157 @@
+# ZERO TRUST SECURITY (HYBRID MODE)
+# =============================================================================
+# OPTION A: ZERO TRUST (RECOMMENDED)
+# Set this token to inject secrets directly from Bitwarden at runtime.
+# If set, the app will ignore local secret variables and fetch them securely.
+# BWS_ACCESS_TOKEN=<place_your_machine_account_token_here>
 
+# Bitwarden Secret IDs (Map these to UUIDs in your Bitwarden Project)
+# The app looks for these IDs if BWS_ACCESS_TOKEN is present.
+LINEAR_WEBHOOK_SECRET_PRD_ID=<uuid>
+POSTGRES_PASSWORD_PRD_ID=<uuid>
+NEO4J_PASSWORD_PRD_ID=<uuid>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEO4J_PASSWORD_PRD_ID key should go before the POSTGRES_PASSWORD_PRD_ID key

(UnorderedKey)